src/main/java/eightbit/task/Task.java

```diff
@@ -0,0 +1,55 @@
+package eightbit.task;
+
+/**
+ * Represents a task that can be completed.
+ */
+public class Task {
```

Comment:
Should this class be abstract, because it's only instantiated in the forms of Todo, Event, etc?
```suggestion
public abstract class Task {
```

Reply:
Thanks for your suggestions!